randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -50,13 +50,15 @@ Extra - room_id: [0]
           Screw Attack
           Any of the following:
               # Break single bomb blocks
-              Can Use Power Bombs
               Screw Attack and Knowledge (Beginner)
-              # TODO: vid of this
+              # Jump Bomb Jump to break the blocks: https://youtu.be/mZe9Ht3o19w
               Hi-Jump and Jump Bombjump (Expert) and Can Use Bombs
+              # Break the blocks with Power Bombs: https://youtu.be/A89giARXvPE
```

Comment:
this isnt a trick, it doesnt *need* a video

Reply:
Removed video from logic db.